src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -3,70 +3,141 @@
+              {% else %}
+                <li class="course-detail__content__subjects__item--draft">
+                  {{ subject.extended_object.get_title }}
+                </li>
+              {% endif %}
+            {# If the current course page is the published version, show only the subjects that are published #}
+            {% elif subject.public_extension %}
+              <li class="course-detail__content__subjects__item">
+                {{ subject.public_extension.extended_object.get_title }}
+              </li>
+            {% endif %}
+          {% empty %}
+            <li class="course-detail__content__subjects--empty">
+              {% trans "No associated subjects" %}
             </li>
-          {% endif %}
-        {# If the current course page is the published version, show only the subjects that are published #}
-        {% elif subject.public_extension %}
-          <li class="course-detail__subjects__item">
-            {{ subject.public_extension.extended_object.get_title }}
+          {% endfor %}
+        </ul>
+      </div>
+
+      <div class="course-detail__content__organizations">
+        <h2>{% trans 'Organizations' %}</h2>
+        <ul class="course-detail__content__organizations__list">
+          {% if course.organization_main %}
+          <li class="course-detail__content__organizations__item course-detail__content__organizations__item--main">
+            {{ course.organization_main.extended_object.get_title }}
           </li>
-        {% endif %}
-      {% empty %}
-        <li class="course-detail__subjects--empty">
-          {% trans "No associated subjects" %}
-        </li>
-      {% endfor %}
-    </ul>
-
-    <ul class="course-detail__organizations">
-      {% for organization in course.organizations.all %}
-        {# If the current page is a draft, show draft organizations with a class annotation for styling #}
-        {% if current_page.publisher_is_draft %}
-          {% if organization.public_extension %}
-            <li class="course-detail__organizations__item">
-              {{ organization.public_extension.extended_object.get_title }}
-            </li>
-          {% else %}
-            <li class="course-detail__organizations__item--draft">
-              {{ organization.extended_object.get_title }}
-            </li>
           {% endif %}
-        {# If the current page is the published version, show only the organizations that are published #}
-        {% elif organization.public_extension %}
-          <li class="course-detail__organizations__item">
-            {{ organization.public_extension.extended_object.get_title }}
-          </li>
-        {% endif %}
-      {% endfor %}
-    </ul>
+          {% for organization in course.organizations.all %}
+            {# If the current page is a draft, show draft organizations with a class annotation for styling #}
+            {% if current_page.publisher_is_draft %}
+              {% if organization.public_extension %}
```

Comment:
you need to wrap all this in one more {% if %} to skip the main organization as it is already displayed separately and is always contained in course.organizations as well.